### PR TITLE
Adding kml_util to documentation index for distro

### PR DESCRIPTION
**Pull request by silviomaeta**
I'd like to have kml_util indexed and documented in ros.org site.

### CODE REVIEW

**Comment by silviomaeta** [comment]
I'd like to have plot_util indexed and documented in ros.org site.

**Comment by silviomaeta** [comment]
I'd like to have kml_util and plot_util indexed and documented in ros.org site.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -4221,6 +4221,12 @@ repositories:
       url: https://github.com/kf/kingfisher.git
       version: indigo-devel
     status: maintained
+  kml_util:
+    doc:
+      type: git
+      url: https://github.com/silviomaeta/kml_util.git
+      version: indigo-devel
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: There is no such branch name in your repo. This should probably be `master`?

Same below.

Comment (silviomaeta) [inline indigo/distribution.yaml]: Yes.  You are right.   It is master.  I will fix it and submit the request again. 
Thanks,
Silvio

Comment (dirk-thomas) [inline indigo/distribution.yaml]: You can also edit the files on the branch of this PR.

**Comment by dirk-thomas** [comment]
After updating the PR you might want to comment in the future. Otherwise nobody is being notified.

**Pull request merged**